modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -15,32 +15,58 @@ You must specify a storage backend in the image set configuration file. This sto
 Do not delete or modify the metadata that is generated by the oc-mirror plug-in. You must use the same storage backend every time you run the oc-mirror plug-in for the same mirror registry.
 ====
 
+.Prerequisites
+
+* You have created a container image registry credentials file. For instructions, see _Configuring credentials that allow images to be mirrored_.
+
 .Procedure
 
-. Create an `ImageSetConfiguration` resource that specifies the necessary configuration details:
+. Use the `oc mirror init` command to create a template for the image set configuration and save it to a file called `imageset-config.yaml`:
++
+[source,terminal]
+----
+$ oc mirror init --registry example.com/example/oc-mirror > imageset-config.yaml <1>
+----
+<1> Replace `example.com/example/oc-mirror` with the location of your registry for the storage backend.
+
+. Edit the file and adjust the settings as necessary:
 +
-.Example `ImageSetConfiguration` file
 [source,yaml]
 ----
-apiVersion: mirror.openshift.io/v1alpha1
 kind: ImageSetConfiguration
-archiveSize: 4 <1>
+apiVersion: mirror.openshift.io/v1alpha2
+archiveSize: 4                                                      <1>
+storageConfig:                                                      <2>
+  registry:
+    imageURL: example.com/example/oc-mirror                         <3>
+    skipTLS: false
 mirror:
- ocp:
-   channels:
-     - name: stable-4.9 <2>
- operators:
-   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9 <3>
-storageConfig: <4>
- registry:
-   imageURL: example.com/example/oc-mirror <5>
+  platform:
+    channels:
+    - name: stable-0.2                                              <4>
+      type: ocp
+    graph: true                                                     <5>
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11  <6>
+    packages:
+    - name: serverless-operator                                     <7>
+      channels:
+      - name: stable
+  additionalImages:
+  - name: registry.redhat.io/ubi8/ubi:latest                        <8>
+  helm: {}
 ----
-<1> The maximum size, in GiB, of each file within the image set.
-<2> The channel to retrieve the {product-title} images from.
-<3> The Operator catalog to retrieve the {product-title} images from.
-<4> The back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values.
-<5> The registry URL for the storage backend.
+<1> Add `archiveSize` to set the maximum size, in GiB, of each file within the image set.
+<2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values.
+<3> Set the registry URL for the storage backend.
+<4> Set the channel to retrieve the {product-title} images from.
+<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for automatic updates.
```

Comment:
@jpower432 You mentioned in the release notes PR that we could go into more detail about OSUS in the oc-mirror docs. This is the only place where I really mention it now. Did you have any thoughts on if we need to go into any more detail on this somewhere else?

Reply:
Do you think we could link to the disconnected OSUS documentation to allow the user to get more information on what the manual process is?

Reply:
We have an annoying limitation right now where I can't actually link to it, but I can do a workaround to mention the title of the section to check out, and link to it in a section below this module. So I'll do that at least